src/resolve/resolved-conf.c

```diff
@@ -128,6 +130,100 @@ int manager_parse_search_domains_and_warn(Manager *m, const char *string) {
         }
 }
 
+int manager_init_dnssec_algorithms(Manager *m) {
+        int r;
+
+        assert(m);
+
+        r = set_ensure_allocated(&m->dnssec_algorithms, NULL);
```

Comment:
don't use that anymore. instead allocate the set on-the-fly when you add the first entry. i.e. use set_ensure_put()